event-adapter/adapter.yaml

```diff
@@ -0,0 +1,134 @@
+# TODO(kawych): fix namespaces
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: event-adapter
+  namespace: default
+---
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: event-adapter
+  namespace: default
+  labels:
+    run: event-adapter
+    k8s-app: event-adapter
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      run: event-adapter
+      k8s-app: event-adapter
+  template:
+    metadata:
+      labels:
+        run: event-adapter
+        k8s-app: event-adapter
+        kubernetes.io/cluster-service: "true"
+    spec:
+      containers:
+        - image: gcr.io/erocchi-gke-dev-1attempt/event-adapter:1.0
+          imagePullPolicy: Always
+          name: pod-event-adapter
+          command:
+            - /adapter
+            - --requestheader-client-ca-file=/var/run/secrets/kubernetes.io/serviceaccount/ca.crt
+          resources:
+            limits:
+              cpu: 250m
+              memory: 200Mi
+            requests:
+              cpu: 250m
+              memory: 200Mi
+---
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    run: event-adapter
+    k8s-app: event-adapter
+    kubernetes.io/cluster-service: 'true'
+    kubernetes.io/name: Adapter
+  name: event-adapter
+  namespace: default
+spec:
+  ports:
+  - port: 443
+    protocol: TCP
+    targetPort: 443
+  selector:
+    run: event-adapter
+    k8s-app: event-adapter
+  type: ClusterIP
+---
+apiVersion: rbac.authorization.k8s.io/v1alpha1
```

Comment:
RBAC is v1beta1 in Kubernetes 1.7 and will probably go to v1 soon. I'd use v1beta1 here. Same in other places.